game/src/main/kotlin/gg/rsmod/game/model/container/ItemContainer.kt

```diff
@@ -13,21 +13,6 @@ import mu.KLogging
  * @author Tom <rspsmods@gmail.com>
  */
 class ItemContainer(val definitions: DefinitionSet, val key: ContainerKey) : Iterable<Item?> {
-    fun toUnnotedOrSelf(item: Item): Item {
-        val unnoted = item.getDef(definitions).realId
-        return if (unnoted == -1) {
-            item
-        } else {
-            Item(unnoted, item.amount)
-        }
-    }
-
-    fun sequence(): Sequence<Item> {
```

Comment:
I moved the sequence function to the bottom to keep the class organized. I also added KDoc comment to explain what the function does and additional information about the functions behavior, and the assumption that an existing iterator method is required.